packages/@o3r/rules-engine/src/engine/ruleset-executor.ts

```diff
@@ -112,8 +140,8 @@ export class RulesetExecutor {
       return undefined;
     } else if (isOperandFact(operand)) {
       const factValue = factsValue[operand.value];
-      // eslint-disable-next-line new-cap
-      return operand.path ? factValue && JSONPath({wrap: false, json: factValue, path: operand.path}) : factValue;
+      // eslint-disable-next-line new-cap -- convention for JSONPath
```

Comment:
can be renamed in `import` instead

Reply:
Should not be a problem anymore after the rebase

Reply:
It would be easier to identify without renaming 
I prefer the justification
because someone could remove the rename at some point 
or if we change the eslint config the rename will be useless

Reply:
Indeed, but this is also one of the purposes of the renaming: giving a name that respect your conventions.
So, if it would have been a new dev, we would have renamed so it would not shocked me to do the same here.
That being said, I do not have a strong opinion on it

Reply:
I don;t like to have extra code to satisfy the linter. I would keep the disable part.